xml/http_servers.xml

```diff
@@ -4246,4 +4246,31 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:acme:mini_httpd:-"/>
   </fingerprint>
 
-</fingerprints>
\ No newline at end of file
+  <fingerprint pattern="^Unspecified, UPnP/[\d\.]+, Unspecified$">
+    <description> UPNP server</description>
```

Comment:
Remove the leading whitespace in the description.

Reply:
completed